modules/page_object_about_profiles.py

```diff
@@ -7,3 +10,58 @@ class AboutProfiles(BasePage):
     """
 
     URL_TEMPLATE = "about:profiles"
+
+    def create_new_profile(self, util: Utilities, ba: BrowserActions) -> BasePage:
+        """
+        Creates a new profile with a random name
+        """
+        pass
+        # profile = Profile()
+        # self.get_element("create-profile-button").click()
+
+        # profile_name = util.generate_random_text("word")
+
+        # current_platform = platform.system()
+
+        # try:
+        #     alert = self.driver.switch_to.alert
+        #     alert_text = alert.text
+        #     print(f"Alert detected with message: {alert_text}")
+        #     alert.accept()  # or alert.dismiss()
+        # except NoAlertPresentException:
+        #     print("No alerts detected.")
+
+        # sleep(0.5)
+        # if current_platform == "Linux":
+        #     self.system_dialog_interation_linux()
+        # elif current_platform == "Windows":
+        #     self.system_dialog_interation_windows()
+        # else:
+        #     print("running")
+        #     self.system_dialog_interation_macos(profile_name, ba)
+        # sleep(10)
+
+    def system_dialog_interation_linux(self):
+        pass
+
+    def system_dialog_interation_macos(self, profile_name: str, ba: BrowserActions):
```

Comment:
Um, I think this might be some leftover debugging stuff...

Reply:
Yeah, forgot to delete!